cartography/models/kandji/device.py

```diff
@@ -0,0 +1,48 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class KandjiDeviceNodeProperties(CartographyNodeProperties):
+    id: PropertyRef = PropertyRef('id')
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+    device_id: PropertyRef = PropertyRef('device_id')
+    device_name: PropertyRef = PropertyRef('device_name')
+    last_check_in: PropertyRef = PropertyRef('last_check_in')
+    model: PropertyRef = PropertyRef('model')
+    os_version: PropertyRef = PropertyRef('os_version')
+    platform: PropertyRef = PropertyRef('platform')
+    serial_number: PropertyRef = PropertyRef('serial_number', extra_index=True)
+
+
+@dataclass(frozen=True)
+class KandjiTenantToKandjiDeviceRelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+# (:KandjiTenant)<-[:DEVICE]-(:KandjiDevice)
```

Comment:
Since the relationship label is called `DEVICE`, the convention is to point _to the device_. Like this:

```
(:KandjiTenant)-[:DEVICE]->(:KandjiDevice)
```

So you would do `LinkDirection.INWARD`.

Reply:
Apologies for the confusion Alex!
The correct relationship is
```
(:KandjiDevice)-[:ENROLLED_TO]->(:KandjiTenant)
```
I had updated the doc but forgot to update the comment (fixed now).

Does the relationship makes sense now?